docs/release-notes/artifacts/pr0257.yaml

```
@@ -0,0 +1,15 @@
+schema_version: 1
+changes:
+  - title: Remove haproxy_route restriction for expressions
+    author: javierdelapuente
+    type: minor
+    description: |
+      Currently some valid character are not allowed for expressions in the
+      haproxy_route relation. Only restrict the newline character.
```

```suggestion
      Currently some valid characters are not allowed for expressions in the
      `haproxy_route` relation. Now only the newline character is restricted.
```
If I misunderstood what this change is doing, let me know!